raft-tech/TANF-app
# 2688 tanf section 2 validation clean up

## Summary of Changes

Pull request closes #2688

This is a cleanup for section 2 validators. Required changes are listed [here](https://gorafttech.sharepoint.com/:x:/r/sites/TDRSResearchDesign/_layouts/15/Doc.aspx?sourcedoc=%7BFB4CFF15-BE50-4BF8-ADCC-7CC16A9AB128%7D&file=TDP_Validation_TANF_Section2.xlsx&action=default&mobileredirect=true)

## How to Test

These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Upload test file
1. Confirm the fields as described in [here](https://gorafttech.sharepoint.com/:x:/r/sites/TDRSResearchDesign/_layouts/15/Doc.aspx?sourcedoc=%7BFB4CFF15-BE50-4BF8-ADCC-7CC16A9AB128%7D&file=TDP_Validation_TANF_Section2.xlsx&action=default&mobileredirect=true) can include space

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] space-fill accounted for tanf section 2
+ [x] sub housing range updated for tanf section 2

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
